packages/@azure/eventhubs/client/lib/connectionContext.ts

```diff
@@ -116,6 +144,10 @@ export namespace ConnectionContext {
         numSenders: Object.keys(connectionContext.senders).length,
         numReceivers: Object.keys(connectionContext.receivers).length
       };
+
+      // Remove all the senders link, receivers link, cbs session and management session for this AMQP connection.
+       remove(connectionContext);
```

Comment:
The name `remove` is not intuitive. At a quick glance, this gives the impression that the `connectionContext` is being removed. How about `clearConnectionLinks`?